oeps/architectural-decisions/oep-0023-style-customization.rst

```diff
@@ -202,7 +202,7 @@ References
       https://openedx.atlassian.net/wiki/spaces/FEDX/pages/112001431/Challenges+with+comprehensive+theming+in+Eucalyptus
 
 7. OEP-16: Bootstrap Adoption
-      https://github.com/edx/open-edx-proposals/blob/a0c284c6c5f0f9419df1f5d6730b2a0c8ff26efc/oeps/oep-0016-bp-adopt-bootstrap.rst
+      https://github.com/openedx/open-edx-proposals/blob/a0c284c6c5f0f9419df1f5d6730b2a0c8ff26efc/oeps/oep-0016-bp-adopt-bootstrap.rst
 
 8. edx-bootstrap edx theme
-      https://github.com/edx/edx-bootstrap/tree/master/sass/edx
+      https://github.com/openedx/edx-bootstrap/tree/master/sass/edx
```

Comment:
Probably irrelevant to this PR, but this 404s. The sass directory was deleted at some point.

Reply:
Happy to fix this - would this be the proper replacement? https://github.com/openedx/edx-bootstrap/tree/master/scss